macos workflows are failing

The macos workflows started failing a few days ago. In the authoritative fork, 'develop' (2a07b2c) was working fine as of August 9th. Sometime after that date, the workflow started failing with an error about a missing jpeg library.

`make[2]: *** No rule to make target /usr/local/lib/libjpeg.dylib, needed by sorc/emcsfc_ice_blend.fd/emcsfc_ice_blend.  Stop.`

If I update the `linux-mac-nceplibs-mpi.yml` workflow file as follows, it works:

```
--- a/.github/workflows/linux-mac-nceplibs-mpi.yml
+++ b/.github/workflows/linux-mac-nceplibs-mpi.yml
@@ -32,7 +32,6 @@ jobs:
         elif [[ ${{ matrix.os }} == "macos-latest" ]]; then
             brew update
           brew install wget
+         brew install jpeg-turbo
           if [[ ${{ matrix.compiler }} == "gcc-9" ]]; then
             sudo ln -sf /usr/local/bin/gfortran-9 /usr/local/bin/gfortran
           elif  [[ ${{ matrix.compiler }} == "gcc-10" ]]; then
```

I am not sure this is the proper solution. @edwardhartnett - any ideas?

--- a/.github/workflows/linux-mac-nceplibs-mpi.yml
+++ b/.github/workflows/linux-mac-nceplibs-mpi.yml
@@ -30,7 +30,7 @@ jobs:
           sudo apt-get install libpng-dev
           sudo apt-get install libjpeg-dev
         elif [[ ${{ matrix.os }} == "macos-latest" ]]; then
-            brew update
+          brew update
           brew install wget
           brew install jpeg-turbo
           if [[ ${{ matrix.compiler }} == "gcc-9" ]]; then

--- a/.github/workflows/linux-mac-nceplibs-mpi.yml
+++ b/.github/workflows/linux-mac-nceplibs-mpi.yml
@@ -32,6 +32,7 @@ jobs:
         elif [[ ${{ matrix.os }} == "macos-latest" ]]; then
             brew update
           brew install wget
+          brew install jpeg-turbo
           if [[ ${{ matrix.compiler }} == "gcc-9" ]]; then
             sudo ln -sf /usr/local/bin/gfortran-9 /usr/local/bin/gfortran
           elif  [[ ${{ matrix.compiler }} == "gcc-10" ]]; then

